README.md

```diff
@@ -573,6 +573,12 @@ certain environment variables:
     - Default Value: `False`
     - Environment Variable Override: `PYTHON_IRODSCLIENT_CONFIG__LEGACY_AUTH__PAM__FORCE_USE_OF_DEDICATED_PAM_API`
 
+-   Setting: Default number of rows for a Query to retrieve by default.
```

Comment:
I think we could use the word "limit" or "maximum" in this description.

Also, I'm a little confused by the word "default" here. This is a configuration value to affect the maximum number of rows that a query can retrieve at a time, right?

Reply:
Admittedly, that description seems wrong, so yes let's change it.
I wasn't actually that familiar with the query limit and its function, so I needed to reconsider this.

Reply:
@alanking please review whether the change i've made in README is appropriate / sufficient ? thanks

Reply:
Looks good to me.